### PR TITLE
Contact Form: adjust wording for filter description.

### PR DESCRIPTION
**Pull request by jeherve**
Follow-up of #20912

#### Changes proposed in this Pull Request:
﻿One can use this filter to automatically set up cc or bcc for all contact forms on the site, based on data submitted by site visitors.
See the description in #20912 for an example.

Let's make it clear that by doing so, you're allowing visitors to send emails to any email address they want.

#### Jetpack product discussion

* Internal reference: p1630937495259700-slack-CBG1CP4EN

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Check for typos.
* Is the explanation clear?

### CODE REVIEW

File: projects/plugins/jetpack/modules/contact-form/grunion-contact-form.php [hunk 1]
```diff
@@ -2889,6 +2889,10 @@ function process_submission() {
 		/**
 		 * Allow customizing the email headers.
 		 *
+		 * Note: use with caution.
+		 * If you use this filter to send emails to specific addresses set by form data,
+		 * you could open your site for abuse, as one could submit random email addresses to email them.
```
Comment (anomiex) [inline projects/plugins/jetpack/modules/contact-form/grunion-contact-form.php:2894]: This doesn't seem dire enough. Maybe something like this?
```suggestion
		 * Warning: DO NOT add headers or header data from the form submission without proper
		 * escaping and validation, or you're liable to allow abusers to use your site to send spam.
		 *
		 * Especially DO NOT take email addresses from the form data to add as CC or BCC headers
		 * without strictly validating each address against a list of allowed addresses.
```

**Review by anomiex — APPROVED** [review]
I'm going to hit "approve" because I don't see any actual problem with the added wording. But I'm also going to suggest making the wording more forceful, see inline.

**Review by kbrown9 — APPROVED** [review]
LGTM!

**Review by anomiex — APPROVED** [review]

**Comment by jeherve** [comment]
r231432-wpcom

**Pull request merged**